pkg/conf/model.go

```diff
@@ -32,25 +47,14 @@ type NodeConfig struct {
 	//      on the host ports 5433 and 9201 respectively
 	HostPorts string `yaml:"hostPorts,omitempty"`
 
-	// One of "current", "prerelease", or an audius-docker-compose git branch (for dev purposes)
-	// "current" corresponds to main adc branch
-	// "prelease" corresponds to stage
-	// defaults to "current" behavior if omitted
-	Version string `yaml:"version,omitempty"`
-
-	PrivateKey    string `yaml:"privateKey"`
-	Wallet        string `yaml:"wallet"`
-	RewardsWallet string `yaml:"rewardsWallet"`
-	EthWallet     string `yaml:"ethWallet,omitempty"`
-
-	// (Optional) Remote content storage
+	// Remote content storage
 	StorageUrl         string `yaml:"storageUrl,omitempty"`
 	StorageCredentials string `yaml:"storageCredentials,omitempty"`
 
-	// (Optional) Postgres db url for remote db and/or custom password
+	// Postgres db url for remote db and/or custom password
```

Comment:
nice, I think there may be some SPs using hosted dbs. blockdaemon nodes for example only have 91gigs on disk which doesn't make sense considering the size of the discovery db + acdc